SportsHub/SportsHub.Api/Program.cs

```diff
@@ -54,6 +54,7 @@
 builder.Services.AddMvc();
 builder.Services.AddControllers();
 builder.Services.AddValidatorsFromAssemblyContaining<UserValidator>();
+builder.Services.AddValidatorsFromAssemblyContaining<CommentValidator>();
```

Comment:
is it from a different assembly than UserValidator?

Reply:
No. removed it